src/main/java/org/fundacionjala/coding/dennis/Twist37.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.Arrays;
+
+/**
+ *
+ */
+public final class Twist37 {
+    /**
+     *
+     */
+    private Twist37() {
+    }
+
+    /**
+     *
+     * @param array Arreglo de enteros
+     * @return Integer[]
+     */
+    public static Integer[] sortTwisted37(final Integer[] array) {
+        Integer[] tw = twist(array.clone());
+        Arrays.sort(tw);
+        return twist(tw);
+    }
+
+    /**
+     *
+     * @param iArray Arreglo de enteros
+     * @return Arreglo
+     */
+    public static Integer[] twist(final Integer[] iArray) {
+        StringBuilder[] sbArray = new StringBuilder[iArray.length];
+        for (int pt = 0; pt < sbArray.length; pt++) {
+            sbArray[pt] = new StringBuilder(iArray[pt].toString()
+                    .replaceAll("3", "a")
+                    .replaceAll("7", "3")
+                    .replaceAll("a", "7"));
+                    iArray[pt] = Integer.parseInt(sbArray[pt].toString());
+        }
+        return iArray;
```

Comment:
This may be refactored like this.
```java
    private static Integer[] twist(final Integer[] iArray) {
        for (int pt = 0; pt < iArray.length; pt++) {
            String twist = String.valueOf(iArray[pt])
                    .replaceAll("3", "a")
                    .replaceAll("7", "3")
                    .replaceAll("a", "7");
            iArray[pt] = Integer.parseInt(twist);
        }
        return iArray;
    }
```